File: filedata/entity/c#/Vipuser.cs

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace Coldairarrow.Entity.Video
{
    /// <summary>
    /// 
    /// </summary>
    [Table("vip_user")]
    public class Vipuser
    {
        
        /// <summary>
        /// ID 主键
        /// </summary>
        [Column("Id")]
        public string Id { get; set; }
    
    
        /// <summary>
        /// 会员站点
        /// </summary>
        [Column("CompanyStyle")]
        public string CompanyStyle { get; set; }
    
    
        /// <summary>
        /// 会员账号
        /// </summary>
        [Column("UserName")]
        public string UserName { get; set; }
    
    
        /// <summary>
        /// 会员昵称
        /// </summary>
        [Column("OtherName")]
        public string OtherName { get; set; }
    
    
        /// <summary>
        /// 会员头像
        /// </summary>
        [Column("HeadImg")]
        public string HeadImg { get; set; }
    
    
        /// <summary>
        /// 会员登录密码 默认123456
        /// </summary>
        [Column("UserPass")]
        public string UserPass { get; set; }
    
    
        /// <summary>
        /// 会员余额
        /// </summary>
        [Column("Balance")]
        public decimal Balance { get; set; }
    
    
        /// <summary>
        /// 会员状态 0=停用 1=启用
        /// </summary>
        [Column("Status")]
        public bool Status { get; set; }
    
    
        /// <summary>
        /// 是否绑定手机
        /// </summary>
        [Column("IsBindPhone")]
        public bool IsBindPhone { get; set; }
    
    
        /// <summary>
        /// 会员手机号
        /// </summary>
        [Column("TelePhone")]
        public string TelePhone { get; set; }
    
    
        /// <summary>
        /// 会员上级账号
        /// </summary>
        [Column("ParentId")]
        public string ParentId { get; set; }
    
    
        /// <summary>
        /// 会员邀请码
        /// </summary>
        [Column("InviteCode")]
        public string InviteCode { get; set; }
    
    
        /// <summary>
        /// 会员注册IP
        /// </summary>
        [Column("AddIP")]
        public string AddIP { get; set; }
    
    
        /// <summary>
        /// 会员注册地址
        /// </summary>
        [Column("AddIPStr")]
        public string AddIPStr { get; set; }
    
    
        /// <summary>
        /// 会员注册时间
        /// </summary>
        [Column("AddTime")]
        public DateTime AddTime { get; set; }
    
    
        /// <summary>
        /// 会员最后登录IP
        /// </summary>
        [Column("LoginIp")]
        public string LoginIp { get; set; }
    
    
        /// <summary>
        /// 会员最后登录地址
        /// </summary>
        [Column("LoginIPStr")]
        public string LoginIPStr { get; set; }
    
    
        /// <summary>
        /// 会员最后登录时间
        /// </summary>
        [Column("LoginTime")]
        public DateTime LoginTime { get; set; }
    
    
        /// <summary>
        /// 登录次数
        /// </summary>
        [Column("LoginCount")]
        public int LoginCount { get; set; }
    
    
        /// <summary>
        /// 绑定银行名
        /// </summary>
        [Column("BankName")]
        public string BankName { get; set; }
    
    
        /// <summary>
        /// 绑定银行卡号
        /// </summary>
        [Column("BankAccount")]
        public string BankAccount { get; set; }
    
    
        /// <summary>
        /// 提款密码
        /// </summary>
        [Column("BandPass")]
        public string BandPass { get; set; }
    
    
        /// <summary>
        /// 是否VIP 0=不是
        /// </summary>
        [Column("IsVip")]
        public bool IsVip { get; set; }
    
    
        /// <summary>
        /// VIP类型 0=普通用户 1=普通VIP用户
        /// </summary>
        [Column("VipType")]
        public int VipType { get; set; }
    
    
        /// <summary>
        /// 会员loginkey
        /// </summary>
        [Column("LoginKey")]
        public string LoginKey { get; set; }
    
    
    }
}

File: filedata/entity/java/vipUser.java

package entity;
// 
public class Vipuser  extends BaseDomain{
    public Vipuser() {

    }
    
    //
    private String Id;
        
    //
    private String CompanyStyle;
        
    //
    private String UserName;
        
    //
    private String OtherName;
        
    //
    private String HeadImg;
        
    //
    private String UserPass;
        
    //
    private double Balance;
        
    //
    private bit(1) Status;
        
    //
    private bit(1) IsBindPhone;
        
    //
    private String TelePhone;
        
    //
    private String ParentId;
        
    //
    private String InviteCode;
        
    //
    private String AddIP;
        
    //
    private String AddIPStr;
        
    //
    private Date AddTime;
        
    //
    private String LoginIp;
        
    //
    private String LoginIPStr;
        
    //
    private Date LoginTime;
        
    //
    private bigint(64) LoginCount;
        
    //
    private String BankName;
        
    //
    private String BankAccount;
        
    //
    private String BandPass;
        
    //
    private bit(1) IsVip;
        
    //
    private int(10) VipType;
        
    //
    private String LoginKey;
        
    public String getId() {
        return Id;
    }
    public void setId(String Id) {
        this.Id = Id;
    }
            
    public String getCompanystyle() {
        return Companystyle;
    }
    public void setCompanystyle(String Companystyle) {
        this.Companystyle = Companystyle;
    }
            
    public String getUsername() {
        return Username;
    }
    public void setUsername(String Username) {
        this.Username = Username;
    }
            
    public String getOthername() {
        return Othername;
    }
    public void setOthername(String Othername) {
        this.Othername = Othername;
    }
            
    public String getHeadimg() {
        return Headimg;
    }
    public void setHeadimg(String Headimg) {
        this.Headimg = Headimg;
    }
            
    public String getUserpass() {
        return Userpass;
    }
    public void setUserpass(String Userpass) {
        this.Userpass = Userpass;
    }
            
    public double getBalance() {
        return Balance;
    }
    public void setBalance(double Balance) {
        this.Balance = Balance;
    }
            
    public bit(1) getStatus() {
        return Status;
    }
    public void setStatus(bit(1) Status) {
        this.Status = Status;
    }
            
    public bit(1) getIsbindphone() {
        return Isbindphone;
    }
    public void setIsbindphone(bit(1) Isbindphone) {
        this.Isbindphone = Isbindphone;
    }
            
    public String getTelephone() {
        return Telephone;
    }
    public void setTelephone(String Telephone) {
        this.Telephone = Telephone;
    }
            
    public String getParentid() {
        return Parentid;
    }
    public void setParentid(String Parentid) {
        this.Parentid = Parentid;
    }
            
    public String getInvitecode() {
        return Invitecode;
    }
    public void setInvitecode(String Invitecode) {
        this.Invitecode = Invitecode;
    }
            
    public String getAddip() {
        return Addip;
    }
    public void setAddip(String Addip) {
        this.Addip = Addip;
    }
            
    public String getAddipstr() {
        return Addipstr;
    }
    public void setAddipstr(String Addipstr) {
        this.Addipstr = Addipstr;
    }
            
    public Date getAddtime() {
        return Addtime;
    }
    public void setAddtime(Date Addtime) {
        this.Addtime = Addtime;
    }
            
    public String getLoginip() {
        return Loginip;
    }
    public void setLoginip(String Loginip) {
        this.Loginip = Loginip;
    }
            
    public String getLoginipstr() {
        return Loginipstr;
    }
    public void setLoginipstr(String Loginipstr) {
        this.Loginipstr = Loginipstr;
    }
            
    public Date getLogintime() {
        return Logintime;
    }
    public void setLogintime(Date Logintime) {
        this.Logintime = Logintime;
    }
            
    public bigint(64) getLogincount() {
        return Logincount;
    }
    public void setLogincount(bigint(64) Logincount) {
        this.Logincount = Logincount;
    }
            
    public String getBankname() {
        return Bankname;
    }
    public void setBankname(String Bankname) {
        this.Bankname = Bankname;
    }
            
    public String getBankaccount() {
        return Bankaccount;
    }
    public void setBankaccount(String Bankaccount) {
        this.Bankaccount = Bankaccount;
    }
            
    public String getBandpass() {
        return Bandpass;
    }
    public void setBandpass(String Bandpass) {
        this.Bandpass = Bandpass;
    }
            
    public bit(1) getIsvip() {
        return Isvip;
    }
    public void setIsvip(bit(1) Isvip) {
        this.Isvip = Isvip;
    }
            
    public int(10) getViptype() {
        return Viptype;
    }
    public void setViptype(int(10) Viptype) {
        this.Viptype = Viptype;
    }
            
    public String getLoginkey() {
        return Loginkey;
    }
    public void setLoginkey(String Loginkey) {
        this.Loginkey = Loginkey;
    }
             
    @Override
    public String toString() {
        return "vip_user[Id=" + Id + ",CompanyStyle=" + CompanyStyle + ",UserName=" + UserName + ",OtherName=" + OtherName + ",HeadImg=" + HeadImg + ",UserPass=" + UserPass + ",Balance=" + Balance + ",Status=" + Status + ",IsBindPhone=" + IsBindPhone + ",TelePhone=" + TelePhone + ",ParentId=" + ParentId + ",InviteCode=" + InviteCode + ",AddIP=" + AddIP + ",AddIPStr=" + AddIPStr + ",AddTime=" + AddTime + ",LoginIp=" + LoginIp + ",LoginIPStr=" + LoginIPStr + ",LoginTime=" + LoginTime + ",LoginCount=" + LoginCount + ",BankName=" + BankName + ",BankAccount=" + BankAccount + ",BandPass=" + BandPass + ",IsVip=" + IsVip + ",VipType=" + VipType + ",LoginKey=" + LoginKey + "]";
    }
        
    
}
        